
# UnitAliveMsg.msg
# Unit -> Observer로 발송하는 Message

# Unit index
int32 index

# unit states
int32 state

